includes/base/controls-stack.php

```diff
@@ -1026,9 +1026,13 @@ public function get_data( $item = null ) {
 	 * @since 2.0.14
 	 * @access public
 	 */
-	public function get_parsed_dynamic_settings( $setting = null ) {
+	public function get_parsed_dynamic_settings( $setting = null, $settings = null ) {
```

Comment:
Who else uses this method?

Reply:
![image](https://user-images.githubusercontent.com/10234691/84117874-de24cc00-aa3a-11ea-9b6f-746914bb1cea.png)